map/util/lanelet2_map_preprocessor/launch/transform_maps.launch.xml

```diff
@@ -4,23 +4,13 @@
   <arg name="pcd_map_path" default=""/>
   <arg name="llt_output_path" default=""/>
   <arg name="pcd_output_path" default=""/>
-  <arg name="x" default="0.0"/>
-  <arg name="y" default="0.0"/>
-  <arg name="z" default="0.0"/>
-  <arg name="roll" default="0.0"/>
-  <arg name="pitch" default="0.0"/>
-  <arg name="yaw" default="0.0"/>
+  <arg name="param_file" default="$(find-pkg-share lanelet2_map_preprocessor)/config/transform_maps.param.yaml"/>
 
   <node pkg="lanelet2_map_preprocessor" exec="transform_maps" name="transform_maps" output="screen">
     <param name="llt_map_path" value="$(var llt_map_path)"/>
     <param name="pcd_map_path" value="$(var pcd_map_path)"/>
     <param name="llt_output_path" value="$(var llt_output_path)"/>
     <param name="pcd_output_path" value="$(var pcd_output_path)"/>
```

Comment:
I think we can move these paths into the yaml file as well.

Reply:
@mitsudome-r Thank you. I moved these parameters to param.yaml. Could you review it?